parseProfiler.js

```diff
@@ -0,0 +1,37 @@
+export function parseProfiler(text, data) {
+  const obj = JSON.parse(text);
+  var processStartTime;
+  for (let row of data) {
+    if (row.operation == "Process Start") {
+      processStartTime = row.start
+      break;
+    }
+  }
+
+  for (let thread of obj.threads) {
+    // Loop through all the threads we have profiler info on
+    for (let i = 0; i < thread.markers.length; i++) {
+      let markers = thread.markers;
+      let profileTime = markers.time[i] / 1000;
+      var duration;
```

Comment:
Nit: here and elsewhere, let's not mix `var` and `let`. We should just generally use `let`, as `var` just gets weird.